package.json

```diff
@@ -19,14 +19,14 @@
     "chessboard component",
     "drag and drop"
   ],
-  "main": "dist/index.js",
-  "module": "dist/index.esm.js",
-  "types": "dist/index.d.ts",
+  "main": "package/dist/index.js",
+  "module": "package/dist/index.esm.js",
+  "types": "package/dist/index.d.ts",
   "files": [
-    "dist"
+    "package/dist"
   ],
   "scripts": {
-    "build": "cp ../README.md . && rollup -c",
+    "build": "rollup -c",
```

Comment:
the readme still needs to be copied from the root into the package for when it's published to npm

Reply:
Will do!